storage/storage_transfer/test_create_client.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2015, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import unittest
+
+import create_client
```

Comment:
so flake8 thinks create_client is just a third party module.

I would add the following option to flake8 and put `create_client` (or `utils` if you rename it) as the application import.

```
--application-import-names=create_client,utils
```